libpng/idf_component.yml

```
@@ -0,0 +1,6 @@
+version: "1.6.37"
+description: Portable Network Graphics(png) C library
+url: https://github.com/espressif/idf-extra-components/tree/master/png
+dependencies:
+  idf: ">=4.4"
+  zlip: "1.2.13"
```

Perhaps this was a typo?
```suggestion
  zlib: "1.2.13"
```

Also, does the dependency has to be so strict? Will libpng work with zlib 1.2.12 or 1.2.14? If yes, let's make the requirement more flexible, e.g. "^1.2.13"